src/AST-Core/RBSequenceNode.class.st

```diff
@@ -176,7 +176,7 @@ RBSequenceNode >> addSelfReturn [
 	| node |
 	self lastIsReturn
 		ifTrue: [ ^ self statements last ].
-	node := RBReturnNode value: (RBSelfNode new).
+	node := RBReturnNode value: (RBVariableNode named: 'self').
```

Comment:
We should avoid such strings ('self', 'super'). Pablo introduced something like "RBVariableNode selfNode" in his PR. We need to push it here.

Reply:
My idea was to make sure that people understand that Variables have no meaning before name analysis. We need to be consistent: We can not do it for temps (humanly impossible to know what kind of temp and which offset). Sp why do it for Self?

Reply:
Hmmm "RBVariableNode selfNode" makes sense... but only for self, super and thisContext.

Reply:
Just that there is a duplication of code now. 
But this particular method has already semantics meaning: we return a self. So it needs to build a valid structure.

Reply:
I will update the OR to use #selfNode, superNode and thisContextNode